<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisMoveWidget</class>
 <widget class="QWidget" name="AxisMoveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AxisMoveWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_laserXP">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>镭码X+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_laserYP">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>镭码Y+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_laserYN">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>镭码Y-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_laserXN">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>镭码X-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_carryXP">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>搬运X+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_carryXN">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>搬运X-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_feedZP">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>上料Z+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_feedZN">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>上料Z-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_unloadZP">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>下料Z+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_unloadZN">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>下料Z-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,1,2,1,2">
     <item row="0" column="5">
      <widget class="QComboBox" name="comboBox_move_step"/>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="checkBox_pmove">
       <property name="text">
        <string>点动</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_speed_low">
       <property name="text">
        <string>慢速</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_speed_normal">
       <property name="text">
        <string>中速</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButton_speed_fast">
       <property name="text">
        <string>快速</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>login()</slot>
 </slots>
</ui>
